src/super_gradients/training/transforms/utils.py

```diff
@@ -0,0 +1,132 @@
+from typing import Tuple
+
+import cv2
+import numpy as np
+
+from super_gradients.training.utils.detection_utils import xyxy2cxcywh, cxcywh2xyxy
+
+
+def _rescale_image(image: np.ndarray, target_shape: Tuple[float, float]) -> np.ndarray:
+    """Rescale image to target_shape, without preserving aspect ratio.
+
+    :param image:           Image to rescale. (H, W, C) or (H, W).
+    :param target_shape:    Target shape to rescale to.
+    :return:                Rescaled image.
+    """
+    return cv2.resize(image, dsize=(int(target_shape[1]), int(target_shape[0])), interpolation=cv2.INTER_LINEAR).astype(np.uint8)
+
+
+def _rescale_bboxes(targets: np.array, scale_factors: Tuple[float, float]) -> np.array:
+    """Rescale bboxes to given scale factors, without preserving aspect ratio.
+
+    :param targets:         Targets to rescale (N, 4+), where target[:, :4] is the bounding box coordinates.
+    :param scale_factors:   Tuple of (sy, sx) scale factors to rescale to.
+    :return:                Rescaled targets.
+    """
+
+    targets = targets.astype(np.float32, copy=True)
+
+    sy, sx = scale_factors
+    targets[:, :4] *= np.array([[sx, sy, sx, sy]], dtype=targets.dtype)
+    return targets
+
+
+def _get_center_padding_params(input_size: Tuple[int, int], output_size: Tuple[int, int]) -> Tuple[int, int, Tuple[int, int], Tuple[int, int]]:
+    """Get parameters for padding an image to given output size, in center mode.
+
+    :param input_size:  Size of the input image.
+    :param output_size: Size to resize to.
+    :return:
+        - shift_h:  Horizontal shift.
+        - shift_w:  Vertical shift.
+        - pad_h:    Horizontal padding.
+        - pad_w:    Vertical padding.
+    """
+    pad_h, pad_w = output_size[0] - input_size[0], output_size[1] - input_size[1]
+    shift_h, shift_w = pad_h // 2, pad_w // 2
+    pad_h = (shift_h, pad_h - shift_h)
+    pad_w = (shift_w, pad_w - shift_w)
+    return shift_h, shift_w, pad_h, pad_w
```

Comment:
maybe `pad_top, pad_left, pad_right, pad_bottom` instead `shift_h, shift_w, pad_h, pad_w` ? I feel this has better clarity to which side this offset is applied

Reply:
Changed in 0cb58e216df83afe13f6123edbee9f33d39dba05